// ngx-quill/aot/src/quill-editor.component.ngfactory.ts
/**
 * @fileoverview This file is generated by the Angular template compiler.
 * Do not edit.
 * @suppress {suspiciousCode,uselessCode,missingProperties}
 */
 /* tslint:disable */


import * as import0 from '@angular/core';
import * as import1 from '../../src/quill-editor.component';
import * as import2 from '@angular/forms';
const styles_QuillEditorComponent:any[] = ([] as any[]);
export const RenderType_QuillEditorComponent:import0.RendererType2 = import0.ɵcrt({
  encapsulation: 2,
  styles: styles_QuillEditorComponent,
  data: {}
}
);
export function View_QuillEditorComponent_0(l:any):import0.ɵViewDefinition {
  return import0.ɵvid(0,[
    (l()(),import0.ɵted((null as any),['\n  '])),
    import0.ɵncd((null as any),0),
    (l()(),import0.ɵted((null as any),['\n']))
  ]
  ,(null as any),(null as any));
}
function View_QuillEditorComponent_Host_0(l:any):import0.ɵViewDefinition {
  return import0.ɵvid(0,[
    (l()(),import0.ɵeld(0,(null as any),(null as any),3,'quill-editor',([] as any[]),(null as any),(null as any),(null as any),View_QuillEditorComponent_0,RenderType_QuillEditorComponent)),
    import0.ɵdid(2383872,(null as any),0,import1.QuillEditorComponent,[import0.ElementRef],(null as any),(null as any)),
    import0.ɵprd(2560,(null as any),import2.NG_VALUE_ACCESSOR,(p0_0:any) => {
      return [p0_0];
    },[import1.QuillEditorComponent]),
    import0.ɵprd(2560,(null as any),import2.NG_VALIDATORS,(p0_0:any) => {
      return [p0_0];
    },[import1.QuillEditorComponent])
  ]
  ,(null as any),(null as any));
}
export const QuillEditorComponentNgFactory:import0.ComponentFactory<import1.QuillEditorComponent> = import0.ɵccf('quill-editor',import1.QuillEditorComponent,View_QuillEditorComponent_Host_0,{
  theme: 'theme',
  modules: 'modules',
  readOnly: 'readOnly',
  placeholder: 'placeholder',
  maxLength: 'maxLength',
  minLength: 'minLength',
  required: 'required',
  formats: 'formats',
  bounds: 'bounds'
}
,{
  onEditorCreated: 'onEditorCreated',
  onContentChanged: 'onContentChanged',
  onSelectionChanged: 'onSelectionChanged'
}
,['[quill-editor-toolbar]']);
